kfs-core/src/main/java/edu/arizona/kfs/sys/KFSConstants.java

```diff
@@ -4,5 +4,13 @@
 
     public static final String INVOICE_NUMBER = "Invoice Number";
     public static final String DUPLICATE_INVOICE_QUESTION_ID = "DVDuplicateInvoice";
+    
+
+
+    public static final class Authorization {

```

Comment:
Please move to PurapAuthorizationConstants.PaymentRequestEditMode (create an edu.arizona version of this constants file):

 public static class PaymentRequestEditMode {
        public static final String PURAP_TAX_ENABLED = "purapTaxEnabled";
        ...
       public static String EDIT_VENDOR_ADDR_EDIT_MODE = "editVendorAddress";
    }